eduroam.md

```diff
@@ -9,29 +9,42 @@ layout: default
 Starting after 2022 commencement RIT is replacing the 'RIT' wifi network with a larger, multi-university 'eduroam' network. This can lead to some confusion, especially for anyone not running MacOS or Windows.
 
 ## Purpose of this guide
-This guide aims to provide linux users at RIT with an alternative to the [official RIT helpdesk guidance](https://help.rit.edu/sp?id=kb_article&sysparm_article=KB0040935&sys_kb_id=89e96dc91b57cd90b4c7ffbf034bcb58&spa=1) for users who want more manual control over their connection than is provided by the provided SecureW2 utility that RIT's instructions guide people to.
+This guide aims to provide Linux users at RIT with an alternate to the [official RIT Helpdesk guidance](https://help.rit.edu/sp/?id=kb_article_view&sysparm_article=KB0040935) for users who want more manual control over their connection than is provided by the SecureW2 utility that RIT's instructions guide people to.
+
+## Required Programs
+
+### GUI Setup
+- NetworkManager and a desktop environment that uses it (KDE, GNOME, LXDE, Cinnamon, MATE)
+- Terminal emulator
+- Root/`sudo` permissions
+
+### CLI Setup
+- `wpa_supplicant`, `iwd`, or `nmcli`
+- Terminal emulator
+- Root/`sudo` permissions
+- Some text editor (terminal text editors such as `micro`, `nvim`/`vim`, or `emacs` are easiest for this purpose, but not strictly necessary)
```

Comment:
nano might be good to add to the list of terminal text editors (possibly) instead of one of the higher learning curve ones like (presumably) vim or emacs.

Reply:
That was my initial reasoning for including `micro`. However, I'll update it to include `nano` instead, as that tends to be default-installed, and its slightly more straight-forward.